riocli/usergroup/list.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+import click
+
+from riocli.config import new_client
+from riocli.constants import Colors, Symbols
+from riocli.utils import tabulate_data
+from riocli.utils.spinner import with_spinner
+
+
+@click.command('list')
+@click.pass_context
+@with_spinner(text="Listing usergroups...")
```

Comment:
It's best not to include the spinner in this location. Instead, you can exclude it from the list and inspection commands. The rationale behind this is that the spinner can disrupt the output when piped, which is not ideal for inspection and listing. Additionally, since these operations are not long-poll and the output is nearly instantaneous, a spinner is unnecessary.

Reply:
Understood